### DISCUSSION TITLE
do we have prop to increase the timeout of error message ?

### DISCUSSION THREAD

**Discussion by yaswanthmaddula-sureify** [CarlosNZ/json-edit-react#119] [Q&A]
The error message stays for a while and disappears itself. It would be helpful if there is a prop to increase the same. I found the collapseTimeout but not this. Please let me know if we have any way to configure this already. Thanks.

**Comment by CarlosNZ** [comment]
No, it's currently a hard-coded constant of 2500ms. 

I can easily add this though. I'll try and do it in the next week or so. Will make an issue.

**Comment by CarlosNZ** [comment]
Hi @yaswanthmaddula-sureify -- I've added this in [v1.16.3](https://www.npmjs.com/package/json-edit-react/v/1.16.3)

You can set the time with the prop `errorMessageTimeout`

Thanks for the suggestion
